### ISSUE TITLE
Request: move (public facing) dotnet-install{.sh,.ps1} scripts to this repo

### ISSUE THREAD

**Issue by am11** [dotnet/arcade#5152]
Going by the conversation in this thread: https://github.com/dotnet/sdk/pull/10721, I think either arcade or runtime would be the appropriate repositories for scripts like `dotnet-install.sh`, since very similar scripts are presently maintained and expertise are available.

My interest is to incorporate some low hanging fixes and then make it work for platforms beyond Linux, emulation environments (e.g. Linux chroot on FreeBSD) etc.

It also makes sense since arcade is the primary consumer of dotnet-install; which I believe per-day telemetry results would agree with, considering high number of CI jobs runs using Arcade SDK.

cc @wli3, @jkotas, @janvorli, for your thoughts about this suggestion?

Thank you!

**Comment by jkotas** [comment]
I agree that it would be a good idea to have one master copy of this script.

The public facing version dotnet-install scripts is documented and linked from https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script . What is the process that updates the script at https://dot.net/v1/dotnet-install.sh ?

**Comment by dagood** [comment]
> It also makes sense since arcade is the primary consumer of dotnet-install; which I believe per-day telemetry results would agree with, considering high number of CI jobs runs using Arcade SDK.

I'm skeptical. It's true that usage via the Arcade toolset is defintiely high, but it's a quite public script we've had for a long time. I'm not aware of any telemetry or stats, unfortunately.

> I think either arcade or runtime would be the appropriate repositories for scripts like `dotnet-install.sh`, since very similar scripts are presently maintained and expertise are available.

I don't think dotnet/runtime makes sense, because the script gets SDKs. That infrastructure and endpoint knowledge doesn't belong in dotnet/runtime. (You need a special arg to make it bring down just a Runtime. And on Windows, it can also get a WindowsDesktop Runtime, which is its own repo as well.) Taking that train of thought to the end, the script should be maintained in dotnet/installer. (Formerly dotnet/core-sdk.)

dotnet/arcade makes *some* more sense to me. I don't think there's precedent for maintaining released artifacts directly in this repo, but it would make sense for Arcade to understand the endpoints.

I mentioned that it would make sense for my team take ownership in https://github.com/dotnet/sdk/issues/4237#issuecomment-581507316. This is an acquisition script and needs proper care and maintenance. /cc @dleeapho @nakarnam @leecow @MichaelSimons

**Comment by am11** [comment]
> needs proper care

On the topic of caring, please also note that all the contribution history got lost when these scripts were moved from cli -> toolset -> sdk. I am sure it was not intentional, but someone will hopefully care to migrate the history of `https://github.com/dotnet/cli/tree/master/scripts`, `https://github.com/dotnet/toolset/tree/master/scripts`, `https://github.com/dotnet/cli/tree/master/Documentation`, `https://github.com/dotnet/toolset/tree/master/Documentation/manpages` etc.

**Comment by MichaelSimons** [comment]
Storing release artifacts in dotnet/arcade is concerning to me.  A primary usage scenario for the script (consumer usage) is outside the scope of the dotnet/arcade charter.  Because of this there are unnecessary challenges to overcome.  Examples include developer mindset when making changes (e.g. this isn't only intended for .NET engineering infra), engineering infrastructure to accommodate release artifacts, etc.  The dotnet/installer repo feels like the more better location to me.

**Comment by dleeapho** [comment]
Yes, dotnet/installer repo is a better place for `dotnet-install.sh`, the changes to isolate the setup and packaging in that repo makes it more so. Ownership is another matter altogether.

**Comment by wli3** [comment]
@donJoseLuis

**Comment by am11** [comment]
> The dotnet/installer repo feels like the more better location to me.

@dagood, @MichaelSimons, sounds like a plan. Any guesstimates when will this move happen (preferablly with commit history cli->toolset->sdk->installer)? Are we waiting for some formal decision?

**Comment by wli3** [comment]
@donJoseLuis has the call to move

**Comment by bozturkMSFT** [comment]
We put this in our list and will start working on it soon. Please allow us until Monday to come up with some estimates.

**Comment by garath** [comment]
Triage: Ownership moved to the installer team.

**Comment by dagood** [comment]
Reopening and moving to dotnet/installer to preserve context.
